DISCOVER/conf.py

```diff
@@ -0,0 +1,111 @@
+###############################################################################
+# Auto-generated by `jupyter-book config`
+# If you wish to continue using _config.yml, make edits to that file and
+# re-generate this one.
+###############################################################################
+
+
+
+version = 'dev'
+language = 'en'
+baseurl = 'https://discover-cookbook.numfocus.org'
+
+import json
+import os
+
+# Load language data from languages.json
+language_json_path = os.path.join(os.path.dirname(__file__), '_static', 'languages.json')
+language_data = []
+current_language_name = None
+
+if os.path.exists(language_json_path):
+    with open(language_json_path, 'r', encoding='utf-8') as f:
+        all_languages = json.load(f)
+        
+# Get the current language name 
+current_language_name = next((lang['name_local'] for lang in all_languages if lang['code'] == language), language)
+
+# Filter out hidden languages for the dropdown
+language_data = [lang for lang in all_languages if not lang.get('hidden', False)]
+
+# Create a mapping of language codes to their localized names
+language_names = {}
+for lang in language_data:
+    if "code" in lang and "name_local" in lang:
+        language_names[lang["code"]] = lang["name_local"]
```

Comment:
I think we no longer need `language_names` dict now that we have already defined `current_language_name` as its own variable.